resources/js/views/settings/users/View.vue

```diff
@@ -411,6 +416,7 @@ export default {
         password_confirmation: null,
         user_locale: null,
         bbb_skip_check_audio: false,
+        timezone: null,
```

Comment:
Would be great if we could set the timezone to the default, same with the locale.